### PR TITLE
"Types/Contributor.ts" Added

### PR DESCRIPTION
**Pull request by Debangan202Ok**
@Abbhiishek check it.
https://github.com/Abbhiishek/codecommunitymusic-client/issues/14#issuecomment-1741765437

### CODE REVIEW

File: types/Contrbutor.ts
Comment (Abbhiishek) [inline types/Contrbutor.ts:1]: Well done..

can we make this scalable with a single contributors giving out more issuelinks and prlinks they have worked on. 

We would be building a contributors page i hope these data will help us built the page.

**Review by Abbhiishek — CHANGES_REQUESTED** [review]

File: types/Contrbutor.ts
Comment (Debangan202Ok) [reply types/Contrbutor.ts:1]: Sure! @Abbhiishek 

What should i do next?

Comment (Abbhiishek) [reply types/Contrbutor.ts:1]: I pushed the new schema. Resolve the merge conflict and rest you are good to go

**Review by Debangan202Ok — COMMENTED** [review]
README.md modified

**Comment by Debangan202Ok** [comment]
hello @Abbhiishek, 
please check my new commits

**Review by Abbhiishek — CHANGES_REQUESTED** [review]
Somone else is working with the documentation, If you want you can assit him in this issue #26 

I have defined the types , kindly use that.

**Pull request closed without merging**